cmd/notary/integration_test.go

```diff
@@ -1025,6 +1066,235 @@ func TestClientKeyImportExportRootOnly(t *testing.T) {
 		t, tempDir, server.URL, "gun", target, tempFile.Name())
 }
 
+// Tests importing and exporting keys for all different roles and GUNs
+func TestClientKeyImportExportAllRoles(t *testing.T) {
```

Comment:
Totally non-blocking - this is a pretty awesome and comprehensive positive test!  :D  I was wondering if it would be easier to read (and maybe save you some work) to do the same import/export work just loop over all 4 roles?  That way it'd be easier to see that you're testing importing with pem headers, and importing without pem headers, etc.

Also wondering if it'd make sense to remove the imported key after each one?  Since it imports twice (once with PEM headers and one without), that'd make sure the test wasn't wasn't somehow just reading the bytes from successful previous import.